src/wallet/rpcwallet.cpp

```diff
@@ -445,12 +445,15 @@ UniValue sendtoaddress(const JSONRPCRequest& request)
             "       \"UNSET\"\n"
             "       \"ECONOMICAL\"\n"
             "       \"CONSERVATIVE\"\n"
+            "       \"EXPLICIT\"\n"
+            "9. feerate                (numeric, optional) Fee rate (sat/kb) to use for EXPLICIT mode.\n"
```

Comment:
Mention that `estimate_mode` must be set to (currenlty) `EXPLICIT` to use this value?

Reply:
`to use for EXPLICIT mode` is not enough?